ccd-claim-migration/src/main/java/uk/gov/hmcts/cmc/ccd/migration/services/DataFixHandler.java

```diff
@@ -174,35 +174,24 @@ private void logMessageForCsv(
         CaseDetails details,
         CaseEventDetails lastEventDetails
     ) {
-        logger.info(new StringBuilder("CSV")
-            .append(",")
-            .append(bucketType)
-            .append(",")
-            .append(ccdCase.getPreviousServiceCaseReference())
-            .append(",")
-            .append(ccdCase.getId())
-            .append(",")
-            .append(details.getLastModified())
-            .append(",")
-            .append(lastEventDetails.getEventName())
-            .append(",")
-            .append(lastEventDetails.getCreatedDate())
-            .append(",")
-            .append(lastEventDetails.getUserFirstName())
-            .append(" ")
-            .append(lastEventDetails.getUserLastName())
-            .append(",")
-            .append(hasProgressed)
-            .toString()
+        logger.info("CSV,"
```

Comment:
I am surprised we still have this {{ccd-claim-migration}}, really we should get rid of this but for now this is ok.